src/Uno.UWP/Web/Http/Cookie.cs

```diff
@@ -0,0 +1,35 @@
+﻿#if __WASM__
```

Comment:
No need to change this, but you can use use `Cookie.wasm.cs` instead.

Reply:
I don't know why I didn't 😀 !